kernelci.org/content/en/docs/contrib.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Contributing Guidelines"
+date: 2022-12-08T09:21:56Z
+draft: false
+weight: 2
+---
+
+KernelCI core project is open for contributions. Contributions may consist of
+adding new build, tests and device types as well as features and bugfixes for
+KernelCI core tools.
+The best way to contribute is to send a PR to [kernelci-core](https://github.com/kernelci/kernelci-core).
+When the PR is created, [KernelCI staging](https://kernelci.org/docs/instances/staging)
+instance takes care of updating the [staging.kernelci.org branch](https://github.com/kernelci/kernelci-core/tree/staging.kernelci.org).
+In general the branch is updated every 8h and a limited set of builds and tests
+is run on it. More detailed information about the logic behind staging runs can
+be found [here](https://kernelci.org/docs/instances/staging).
+
+There are several guidelines which can facilitate the PR review process.
+
+1. Make sure the PR is well described
+   1. Describe the purpose of the changes
+   2. Example use cases are welcome
+2. Attach staging build/test results when possible
+   1. Staging instance is updated every 8h. If the PR is expected to produce build/test results
```

Comment:
The first sentence is already explained above -- should it be removed here?

Reply:
The GitHub UI makes my comment hard to understand -- I am referring here to line 24 but GitHub shows context, i.e. in particular I was talking about the "Staging instance is updated every 8h." sentence.

Reply:
I put it there as a reminder, just to make it clear that the author needs to wait at least 8h to see results on staging. On second thought, It may be too much.

Reply:
Yeah, I think either way is fine -- I didn't know if it was intentional. In some types of material, there is value on repeating key information/concepts.

One alternative is saying something shorter that reminds the reader about it, without duplicating information. For instance, something along the lines of "If build/test results are expected to be visible, then link them from staging." (which also allows you to just say "If not" in the next line, if I understand correctly)